test/JDBC/expected/BABEL-EXECUTE_AS_CALLER-vu-prepare.out

```diff
@@ -135,6 +148,21 @@ GO
 ~~ERROR (Message: 'EXECUTE AS SELF|OWNER|<user>|<login>' is not currently supported in Babelfish)~~
 
 
+-- triggers with more than 1 trigger option and with-clause
+CREATE TRIGGER babel_execute_as_caller_trigger2 on babel_execute_as_caller_table WITH EXECUTE AS CALLER AFTER INSERT, DELETE
+AS
+BEGIN
+  SELECT 'babel_execute_as_caller_trigger2 invoked'
+END
+GO
+
+CREATE TRIGGER babel_execute_as_caller_trigger3 on babel_execute_as_caller_table WITH EXECUTE AS CALLER, SCHEMABINDING AFTER INSERT, DELETE
```

Comment:
Let's add `UPDATE` to `dml_trigger_operation` so that we can test different number of commas in `WITH` clause and `dml_trigger_operation`.

Reply:
Fixed in next revision by creating a trigger with trigger option update along with other trigger_options.